components/avatar/theme.css

```diff
@@ -17,8 +17,7 @@
   position: relative;
 }
 
-.bullet,
-.counter {
+.connected {
```

Comment:
Can you explain the name `connected`?

Reply:
I called it `connected` because it's visually connected to the left upper corner of the avatar image. I can call it `.children` if you prefer :)

Reply:
I guess that would be better.